src/hooks/reactQuery/wizards/useMyWizards.ts

```diff
@@ -33,6 +40,18 @@ type UseUpdateWizardProps = {
   updatedWizardDetails: WizardUpdateDetails;
 };
 
+type UseUploadCustomImageProps = {
+  fileName: string;
+  base64Image: string;
+};
+
+export const useFetchImageWizard = ({ userId }: useFetchMyWizardsProps) =>
+  useQuery({
+    queryKey: [QUERY_KEYS.MY_WIZARDS_LIST, userId],
+    queryFn: () => wizard_details.getUserWizards(userId!),
+    enabled: !!userId,
+  });
+
```

Comment:
this is related to images right, it would make sense to have it in `src/hooks/reactQuery/useElnaImages.ts`

Reply:
I'll check the PR and cleanup all thats not necessary before I ask you to review it going forward. This fn was used for a test as well, its not required for anything in this feature.